### PR TITLE
[UE] Implement UE support for "topics" pages

### PR DESCRIPTION
**Pull request by jckautzmann**
- add support for blocks used in topics pages: takeaway, call-to-action, accordion

Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #1083 

Test URLs:
- Before: https://main--danaher-ls-aem--hlxsites.hlx.page/
- After: https://ue-all-blocks--danaher-ls-aem--hlxsites.hlx.page/

Example page:
https://publish-p93411-e849602.adobeaemcloud.com/content/danaher/ls/us/en/products/centrifuges/topics/cell-harvesting-centrifugation-techniques.html

### CODE REVIEW

File: component-models.json [hunk 1]
```diff
@@ -1,4 +1,35 @@
...
+  {
+    "id": "page-metadata",
+    "fields": [
+      {
+        "component": "container",
+        "label": "Fieldset",
+        "fields": [
+          {
+            "component": "text",
+            "name": "canonical",
```
Comment (Buuhuu) [inline component-models.json:11]: that should be the build in `cq:canonicalUrl`

File: component-models.json [hunk 2]
```diff
@@ -1,4 +1,35 @@
...
+            "label": "Creation Date"
+          },
+          {
+            "component": "date-time",
+            "name": "updateDate",
+            "label": "Update Date"
+          },
+          {
+            "component": "text",
+            "name": "FullCategory",
```
Comment (Buuhuu) [inline component-models.json:26]: better name it in camel case `fullCategory`

File: component-models.json [hunk 3]
```diff
@@ -152,5 +183,128 @@
...
+            "value": "h5"
+          },
+          {
+            "name": "h6",
+            "value": "h6"
+          }
+        ]
+      },
+      {
+        "component": "reference",
```
Comment (Buuhuu) [inline component-models.json:243]: that should probably be `aem-content` to link to some content. `reference` is for the asset picker.

File: component-models.json [hunk 4]
```diff
@@ -152,5 +183,128 @@
...
+      {
+        "component": "select",
+        "name": "content_headingType",
+        "value": "h1",
+        "label": "Type",
+        "valueType": "string",
+        "options": [
+          {
+            "name": "h1",
+            "value": "h1"
```
Comment (mhaack) [inline component-models.json:218]: No `h1` here

**Review by mhaack — APPROVED** [review]

**Pull request merged**